Answers.md

```diff
@@ -1,2 +1,41 @@
 # Your responses to the short answer questions should be laid out here using Mark Down.
 ### For help with markdown syntax [Go here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+
+## Question 1
+1. Describe some of the differences between `.forEach` & `.map`.
+
+The difference between forEach and map is that map mutates the original Array while forEach creates a new Array and leaves the original Array un-mutated.
+
+## Question 2
+2. Name five different Types in JavaScript. A Type is something that can represent data. What is so special about Arrays?
+
+Different Types:
+- String
+- Number
+- Boolean
+- Object
+- Function
+
+An Array is special because it is actually an Object under-the-hood.
```

Comment:
I'm looking for a bit more depth here. Maybe something like "they're zero based"